demo_nodes_cpp/src/parameters/even_parameters_node.cpp

```diff
@@ -29,8 +28,11 @@ class EvenParameterNode : public rclcpp::Node
   explicit EvenParameterNode(rclcpp::NodeOptions options)
   : Node("even_parameters_node", options.allow_undeclared_parameters(true))
   {
-    // Force flush of the stdout buffer.
-    setvbuf(stdout, NULL, _IONBF, BUFSIZ);
```

Comment:
Just wanted to check. I know this was here to make sure tests pass. But I also vaguely recall that we've switched to unbuffered logging by default. Is that right?

Reply:
Yeah, that's correct.  In Foxy, we changed to printing logging to stderr by default.  Leaving this line in wouldn't hurt anything, it just isn't necessary anymore.